src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -186,24 +190,21 @@ private void SendEvent<TParams>(IEventSender requestContext, EventType<TParams>
         /// </summary>
         private void RunTask<T>(RequestContext<T> context, ScriptingOperation operation)
         {
-            Task.Run(() =>
```

Comment:
Please undo this. If you need a sync method, create a separate sync method for this operation